tox.ini

```diff
@@ -55,6 +55,14 @@ commands =
     mypy --python-version 3.5 src/build
     mypy --python-version 2.7 src/build
 
+[testenv:{py39, py38, py37, py36, py35, py27, pypy3, pypy2}-minversion]
+description = check minimum versions required of all dependencies
```

Comment:
You want skip_install=True here.

Reply:
Is that better than adding `-c tests/constraints.txt` to the install command?

Reply:
I'd say so.

Reply:
Okay.